pkg/apiserver/registry/datapackaging/package_storage_client.go

```diff
@@ -152,7 +152,28 @@ func (t PackageTranslator) ToExternalWatcher(intObjWatcher watch.Interface, fiel
 }
 
 func (t PackageTranslator) ToExternalError(err error) error {
-	// TODO: implement
+	statusErr, isStatusError := err.(*errors.StatusError)
```

Comment:
I think we can use the APIStatus interface, like [this].(https://github.com/kubernetes/apimachinery/blob/12dc3f82eb47fc972b5be0b222f4422cb1b51612/pkg/api/errors/errors.go#L99)
```
status, ok := err.(APIStatus)
if !ok {
  return err
}
```

Reply:
nit: Typically, we just use an `ok` (instead of `isStatusError`) as it's a very short lived variable.